src/platform-apps/channels/channel-view.tsx

```diff
@@ -79,13 +92,47 @@ export class ChannelView extends React.Component<Properties> {
   render() {
     return (
       <div className='channel-view'>
+        {this.props.hasNewMessage > 0 && (
```

Comment:
can we extract this? this feels like a separate component to me.

Reply:
OK